Cargo.toml

```diff
@@ -2,7 +2,7 @@
 debug = true
 
 [workspace]
-members = [ "jxl", "jxl_cli", "jxl_headers_derive"]
+members = [ "jxl", "jxl_cli", "jxl_headers_derive", "jxlinfo"]
```

Comment:
Why is it not part of `jxl_cli`? You can have more than one binary per crate

Reply:
ok, will put it there

Reply:
moved, now it runs with 
```
cargo run -p jxl_cli --bin jxlinfo [your filename here]
```